apiClient/client.ts

```diff
@@ -20,20 +20,32 @@ import {
 
 export const API_URL = process.env.NEXT_PUBLIC_API_URL
 
-export async function createUser(
-  email: string,
-  graffiti: string,
-  socialChoice: string,
-  social: string,
-  country_code: string,
+type Params = {
+  email: string
+  graffiti: string
+  socialChoice: string
+  social: string
+  country: string
+  recaptcha: string
   github?: string
-): Promise<ApiUser | ApiError> {
+}
+
+export async function createUser({
+  email,
+  graffiti,
+  socialChoice,
+  social,
+  country: country_code,
+  recaptcha,
+  github,
+}: Params): Promise<ApiUser | ApiError> {
```

Comment:
Did this so we don't have to rely on parameter order to build the right payload.